docs/src/algolia-search.js

```diff
@@ -2,9 +2,38 @@ import algoliasearch from 'algoliasearch';
 
 /* eslint no-underscore-dangle: 0 */
 (function () {
+  const algoliaAppId = 'UMBCUJCBE8';
+  const algoliaApiKey = 'b6c4bdb11b865250add6fecc38d8ebdf';
+  const algoliaIndexName = 'yugabytedb_docs';
   const ignoreClickOnMeElement = document.querySelector('body:not(.td-searchpage) .search-area');
   const searchInput = document.getElementById('search-query');
 
+  /**
+   * Decode the cookie and return the appropriate cookie value if found
+   * Otherwise empty string is returned.
+   *
+   * return string
+   */
+  function getCookie(cname) {
```

Comment:
We have cookie processing on 5 different files(index.js, search-banner.js, track-utm, sidebar-tree and this one). Can we have single library to do this?

Reply:
@premkumr Sure, let's do that but it is something not particularly related to the Algolia click events so let's merge this PR and work on cookie processing in a separate PR.